sw/applications/example_spi_write/main.c

```diff
@@ -0,0 +1,176 @@
+/**
+ * @file main.c
+ * @brief Simple spi write example using BSP
+ *
+ * Simple example that writes a 1kB buffer to flash memory at a specific address
+ * and then read it back to check if the data was written correctly.
+ * 
+ * @note The application assume the correct functioning of the read operation.
+ *
+*/
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "x-heep.h"
+#include "w25q128jw.h"
+
+
+// Test functions
+void test_write(uint32_t *test_buffer, uint32_t len);
+void test_write_dma(uint32_t *test_buffer, uint32_t len);
+void test_write_quad(uint32_t *test_buffer, uint32_t len);
+void test_write_quad_dma(uint32_t *test_buffer, uint32_t len);
+
+// Check function
+void check_result(uint32_t *test_buffer, uint32_t len);
+
+
+// Start buffers:
+#include "buffer.bin"
```

Comment:
it is counter intuitive to have that as a bin when it is NOT a bin, it is formated as a normal header, can you change it, please?